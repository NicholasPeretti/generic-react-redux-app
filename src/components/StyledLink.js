import LinkStyle from '../utils/styledComponents/link-style'
import styled from 'styled-components'

const StyledLink = styled.a`
  ${LinkStyle}
`
export default StyledLink
