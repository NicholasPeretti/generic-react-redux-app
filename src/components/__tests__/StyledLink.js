import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import StyledLink from '../StyledLink'

describe('PaginationNav', () => {
  PresentationalTest(StyledLink)

  test('Should match snapshot', () => {
    const wrapper = shallow(<StyledLink/>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
