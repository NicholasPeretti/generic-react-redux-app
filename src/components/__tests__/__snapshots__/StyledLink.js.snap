// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaginationNav Should match snapshot 1`] = `"<a class=\\"sc-bdVaJa cwYlnv\\"></a>"`;
